doc/source/methods/chisquaredrift.ipynb

```diff
@@ -35,7 +35,7 @@
     "\n",
     "* `p_val`: p-value used for significance of the Chi-Squared test for. If the FDR correction method is used, this corresponds to the acceptable q-value.\n",
     "\n",
-    "* `categories_per_feature`: Optional dictionary with as keys the feature column index and as values the number of possible categorical values for that feature. E.g.: *{0: 5, 1: 9, 2: 7}*. If it is not specified, `categories_per_feature` is inferred from `x_ref`.\n",
+    "* `categories_per_feature`: Optional dictionary with as keys the feature column index and as values the number of possible categorical values for that feature. E.g.: *{0: 4, 1: 2, 2: 3}*. Alternatively, for each feature the possible categories can be provided in a list. E.g.: *{0: [0, 1, 2, 3], 1: [0, 1], 2: [0, 1, 2]}*. If it is not specified, `categories_per_feature` is inferred from `x_ref`.\n",
```

Comment:
Same as comment on `tabulardift.ipynb`, expand documentation and make it consistent between the two notebooks.